WooCommerce/Classes/ViewModels/Order Details/OrderPaymentDetailsViewModel.swift

```diff
@@ -111,6 +109,19 @@ final class OrderPaymentDetailsViewModel {
         return String.localizedStringWithFormat(template, styleDate, order.paymentMethodTitle)
     }
 
+    /// Awaiting payment
+    ///
+    var awaitingPayment: String? {
```

Comment:
Nit: The naming can be improved for more clarity 😅 
```suggestion
    var awaitingPaymentTitle: String? {
```

Reply:
Also, we should keep this property `private` since it's needed for this view model only.